apps/src/standaloneVideo/Video.tsx

```diff
@@ -60,11 +71,41 @@ const Video: React.FunctionComponent<VideoProps> = ({children, src}) => {
     height = width / videoAspectRatio;
   }
 
+  const [videoChoice, setVideoChoice] = useState<VideoChoiceType>(undefined);
+
+  const videoCheckComplete = useCallback(
+    (youTubeAvailable: boolean) => {
+      setVideoChoice(youTubeAvailable ? 'youtube' : 'fallback');
+    },
+    [setVideoChoice]
+  );
+
+  useEffect(() => {
+    testYouTubeAvailable(videoCheckComplete);
+  }, [videoCheckComplete]);
```

Comment:
nit: do we need to break out the callback into a separate `useCallback`? thoughts on doing it all in the `useEffect`, i.e.

```
useEffect(() => {
  testYouTubeAvailable(available => setVideoChoice(available ? 'youtube' : 'fallback'));
}, [setVideoChoice]);
```

Reply:
Good call.  Updated.